Gaming The Syllabus/Assets/AnswerSubmissionManager.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// Singleton class that handles the user clicking the button to submit their answer to a riddle. 
+/// <para>Setup: Attach to submit button and set onClick to invoke <see cref="SubmitAnswer"/></para>
+/// </summary>
+public class AnswerSubmissionManager : Singleton<AnswerSubmissionManager>
+{
+    private List<IRiddleAnswerUIElement> selectedAnswers = new();
+
+    private void Awake()
+    {
+        InitializeSingleton();
+    }
```

Comment:
Good call on making this a singleton. I've caught myself a few times forgetting to extend them on classes I create where they are very much needed.